tests/integration/middleware.test.ts

```diff
@@ -130,7 +130,7 @@ for (const {
         expect(response.status).toBe(307)
         expect(response.headers.get('location'), 'added a location header').toBeTypeOf('string')
         expect(
-          new URL(response.headers.get('location') as string).pathname,
+          new URL(response.headers.get('location') as string, 'http://n').pathname,
```

Comment:
Added `base` here because relative `location` without it just fails to construct `URL` object and throws `Invalid URL`. We do check `pathname` anyway so base will be discarded